openedx/features/job_board/tests/views/test_job_board.py

```diff
@@ -0,0 +1,41 @@
+from django.contrib.sites.models import Site
+from django.test import TestCase
+from django.urls import reverse
+from rest_framework import status
+
+from openedx.core.djangoapps.theming.models import SiteTheme
+from openedx.features.job_board.views import JobCreateView, JobListView
+from openedx.features.job_board.tests.factories import JobFactory
+
+
+class JobBoardViewTest(TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super(JobBoardViewTest, cls).setUpClass()
+        site = Site(domain='testserver', name='test')
+        site.save()
+        theme = SiteTheme(site=site, theme_dir_name='philu')
+        theme.save()
```

Comment:
Lets use utility method `configure_philu_theme` from `openedx/core/djangolib/testing/philu_utils.py`

Ignore this comment incase, you do not have that method in your branch, since it was directly added in master

Reply:
I have now added configure_philu_theme since when this will merged with develop branch it will start working